nixos/modules/services/misc/omnom.nix

```diff
@@ -67,6 +67,22 @@ in
                 default = 20;
                 description = "Number of results per page.";
               };
+              template_dir = lib.mkOption {
+                type = lib.types.str;
+                default = "${cfg.dataDir}/templates";
+                defaultText = lib.literalExpression ''
+                  "''${config.services.omnom.dataDir}/templates"
+                '';
+                description = "Where templates are stored.";
+              };
+              static_dir = lib.mkOption {
+                type = lib.types.path;
+                default = "${cfg.dataDir}/static";
```

Comment:
Why not just make it `"${cfg.package}/share/static"`?

Reply:
That's where the directory will be. We can't configure this to `"${cfg.package}/share/static"` because Omnom writes to `static/data`. Instead, we [copy this with tmpfiles](https://github.com/NixOS/nixpkgs/pull/409314/files#diff-bea2407e6c4faab93ae203e0d057587a28fcb3dd03e2f5b288b7709d7286b15aR247) to the `static_dir`, which will only be done once.

Reply:
That's horrifying

Reply:
What about `template_dir`? Does it write there too?

Reply:
> That's horrifying

Yeah, I think that we should iterate over this with upstream and improve it in the future.

> What about `template_dir`? Does it write there too?

No, the `template_dir` is fine.

Reply:
Does setting `settings.app.static_dir` to something other than default value actually makes sense then? If not, I'd remove this option to avoid confusion since it's not really a static directory...